jupyter/intel/ml/ubi9-python-3.9/test/test_notebook.ipynb

```diff
@@ -212,13 +211,6 @@
     "        with self.assertRaises(KafkaConfigurationError):\n",
     "            KafkaConsumer(bootstrap_servers='localhost:9092', api_version=(0, 9), group_id='foo', session_timeout_ms=50000, request_timeout_ms=40000)\n",
     "\n",
-    "class TestKFPTekton(unittest.TestCase):\n",
-    "\n",
-    "    def test_version(self):\n",
-    "        expected_major_minor = '1.6.0'\n",
-    "\n",
-    "        self.assertLess(kfp_tekton.__version__, expected_major_minor)\n",
-    "\n",
```

Comment:
Do we want to remove it also from the intel one? I still can see the `kfp-tekton` in relevant [Pipfile here](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/intel/ml/ubi9-python-3.9/Pipfile#L36)?

Actually, writing this comment now, is it okay that some of our images under the jupyter directory still contains the `kfp-tekton`?

```
$ grep -r kfp-tekton                                                                         
jupyter/intel/pytorch/ubi9-python-3.9/Pipfile.cpu:kfp-tekton = "<1.6.0"
jupyter/intel/pytorch/ubi9-python-3.9/Pipfile.gpu:kfp-tekton = "<1.6.0"
jupyter/intel/pytorch/ubi9-python-3.9/Pipfile.lock.cpu:        "kfp-tekton": {
jupyter/intel/pytorch/ubi9-python-3.9/Pipfile.lock.gpu:        "kfp-tekton": {
jupyter/intel/ml/ubi9-python-3.9/Pipfile:kfp-tekton = "<1.6.0"
jupyter/intel/ml/ubi9-python-3.9/Pipfile.lock:        "kfp-tekton": {
jupyter/intel/tensorflow/ubi9-python-3.9/Pipfile.cpu:kfp-tekton = "<1.6.0"
jupyter/intel/tensorflow/ubi9-python-3.9/Pipfile.gpu:kfp-tekton = "<1.6.0"
jupyter/intel/tensorflow/ubi9-python-3.9/Pipfile.lock.cpu:        "kfp-tekton": {
jupyter/intel/tensorflow/ubi9-python-3.9/Pipfile.lock.gpu:        "kfp-tekton": {
habana/1.10.0/ubi8-python-3.8/Pipfile:kfp-tekton = "<1.6.0"
habana/1.10.0/ubi8-python-3.8/Pipfile.lock:        "kfp-tekton": {
habana/1.11.0/ubi8-python-3.8/Pipfile:kfp-tekton = "<1.6.0"
habana/1.11.0/ubi8-python-3.8/Pipfile.lock:        "kfp-tekton": {
habana/1.9.0/ubi8-python-3.8/Pipfile:kfp-tekton = "<1.6.0"
habana/1.9.0/ubi8-python-3.8/Pipfile.lock:        "kfp-tekton": {
habana/1.13.0/ubi8-python-3.8/Pipfile:kfp-tekton = "<1.6.0"
habana/1.13.0/ubi8-python-3.8/Pipfile.lock:        "kfp-tekton": {
```

If this is something we need for the DSPv2, shouldn't we update this for all supported images? Maybe we don't care about the intel ones for rhoai and we care just about the habana 1.13?

Reply:
Hey, Jan thanks for the review! :slightly_smiling_face: 
I'll revert the change from Intel since this PR was already in place before the removal of kfp_tekton.

Regarding Habana - The latest 1.13 has been updated with the removal [[1]( https://github.com/opendatahub-io/notebooks/blob/main/habana/1.13.0/ubi8-python-3.8/Pipfile#L51)] but the others haven't.
@harshad16 what are your thoughts on what we should do for the rest of Habana?

Reply:
Ah, yeah, sorry about it - forgot to update the local cloned repo :upside_down_face: this is good then.

Regarding the others - let's see what Harshad thinks. Technically, we should care just about the "supported" (not sure what we mean with it in upstream, though) images so they can work with the DSPv2, correct?

Reply:
No changes are required for the intel-based images.
we would work on the intel-based images, as we move further in upcoming sprints.